### PR TITLE
Add a feature to turn off animations for printing as requested.

### PR DESCRIPTION
**Pull request by WarenGonzaga**
Hello, @daneden and @eltonmesquita I just add this feature as requested. This is for this feature request #821 let's make it available for version 4.0. will add more feature as requested.

### CODE REVIEW

**Comment by daneden** [comment]
It might make more sense to add the rule to the media query that already exists for reduced motion

**Comment by WarenGonzaga** [comment]
@daneden you're right!

**Comment by WarenGonzaga** [comment]
hey @daneden they are using different @media querry...

```css

@media (prefers-reduced-motion) {
  .animated {
    animation: unset !important;
    transition: none !important;
  }
}

@media print {
  .animated {
    animation: none!important;
  }
}

```

how I can mix them up?

**Comment by eltonmesquita** [comment]
Hello @WarenGonzaga . The feature is ok, but you missed building it. Can you update the PR so I can merge?

File: source/_base.css [hunk 1]
```diff
@@ -43,7 +43,7 @@
   animation-duration: 3s;
 }
 
-@media (prefers-reduced-motion) {
+@media print and (prefers-reduced-motion) {
```
Comment (daneden) [inline source/_base.css]: This should be a logical `or`, not an `and`:

```css
@media (print), (prefers-reduced-motion) {
  ...
}
```

Comment (WarenGonzaga) [reply source/_base.css]: Awesome! Thanks @daneden

**Pull request merged**